SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -463,14 +460,31 @@ void UGlobalStateManager::AuthorityChanged(const Worker_AuthorityChangeOp& AuthO
 	switch (AuthOp.component_id)
 	{
 		case SpatialConstants::DEPLOYMENT_MAP_COMPONENT_ID:
+		{
 			GlobalStateManagerEntityId = AuthOp.entity_id;
 			SetAcceptingPlayers(true);
 			break;
+		}
 		case SpatialConstants::SINGLETON_MANAGER_COMPONENT_ID:
+		{
 			ExecuteInitialSingletonActorReplication();
 			break;
+		}
+		case SpatialConstants::STARTUP_ACTOR_MANAGER_COMPONENT_ID:
+		{
+			if (bCanBeginPlay)
```

Comment:
nit: Can we invert this, to prevent an additional return mid-function.

Reply:
done